standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/common/IPStackUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.common;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class IPStackUtils {
```

Comment:
We are copying some code over here, can't we keep everything at one place, either of the jars would have dependency of one another. I think hive-common has a dependency of metastore-common, so everything can be kept in metastore-common?
https://github.com/apache/hive/blob/db041fd7b71a15a8bd1a997b0908a02e8124d7ac/common/pom.xml#L46-L49

Reply:
Yes, it's much better to have have everything in one place. I moved the utils class to metastore-common.